Cosmetics.py

```diff
@@ -435,9 +439,9 @@ def patch_gauntlet_colors(rom, settings, log, symbols):
     # patch gauntlet colors
     gauntlets = [
         ('Silver Gauntlets', 'silver_gauntlets_color', 0x00B6DA44,
-            ([0x173B4CC], [0x173B4D4, 0x173B50C, 0x173B514])), # GI Model DList colors
+            ([0x173B4CC], [0x173B4D4, 0x173B50C, 0x173B514], [])), # GI Model DList colors
```

Comment:
Why the extra empty list here and a couple other places seemingly unrelated to the PR?

Reply:
I added a `light_addresses` parameter to the tuple that I attempt to iterate through. For now, `light_addresses` is only used for potion models.